packages/harden/README.md

```diff
@@ -0,0 +1,39 @@
+# harden
+
+For common usage, this package provides an implementation of `harden` that will
+freeze an object and its transitive properties, making them shallowly
```

Comment:
```suggestion
freeze an object and its transitive own properties, making them shallowly
```

Additional pedantic detail that can be omitted in a high level summary, but needs to be explained somewhere. It does a "transitive reflective" walk of properties. The difference is that avoids invoking getters. Rather it proceeds to walk both getters and setters as objects.